backend/YouTubeV2.Application/Constants.cs

```diff
@@ -0,0 +1,12 @@
+﻿namespace YouTubeV2.Application
+{
+    public static class Constants
```

Comment:
maybe add constants folder and there add UserConstants class so it don't get messy